sale_stock_available_info_popup/tests/test_sale_stock_available_info_popup.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from datetime import timedelta
+
+from odoo.fields import Datetime
+from odoo.tests import Form, common, new_test_user
+from odoo.tests.common import users
+
+
+class SaleStockAvailableInfoPopup(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stock_user = new_test_user(
+            cls.env,
+            login="stock_user",
+            groups="sales_team.group_sale_salesman,stock.group_stock_user",
+        )
+        cls.product = cls.env["product.product"].create(
+            {"name": "Storable product", "detailed_type": "product"}
+        )
+        cls.partner = cls.env["res.partner"].create({"name": "Mr. Odoo"})
+        cls.env["stock.quant"].create(
+            {
+                "product_id": cls.product.id,
+                "location_id": cls.env.ref("stock.stock_location_stock").id,
+                "quantity": 40.0,
+            }
+        )
+        picking_in = cls._create_picking(cls, cls.env.ref("stock.picking_type_in"), 5)
+        picking_in.action_assign()
+        picking_out = cls._create_picking(cls, cls.env.ref("stock.picking_type_out"), 3)
+        picking_out.action_assign()
+
+    def _create_picking(self, picking_type, qty):
+        picking_form = Form(self.env["stock.picking"])
+        picking_form.picking_type_id = picking_type
+        with picking_form.move_ids_without_package.new() as move:
+            move.product_id = self.product
+            move.product_uom_qty = qty
+        return picking_form.save()
+
+    def _create_sale(self):
+        sale_form = Form(self.env["sale.order"])
+        sale_form.partner_id = self.partner
+        sale_form.user_id = self.stock_user
+        return sale_form
+
+    def _add_sale_line(self, sale_form, product, qty=1):
+        with sale_form.order_line.new() as line:
+            line.product_id = self.product
+            line.product_uom_qty = qty
+
+    @users("stock_user")
```

Comment:
Don't remove `admin` from both tests. We want to know if it fails due to permissions (the other user) or feature (the admin user).